models/tasks.js

```diff
@@ -631,6 +632,40 @@ const updateTaskStatus = async () => {
   }
 };
 
+const updateOrphanTasksStatus = async (lastOrphanTasksFilteration) => {
+  const lastTimestamp = Number(lastOrphanTasksFilteration);
+  try {
+    const users = [];
+    const currentTimeStamp = Date.now();
+
+    const usersQuerySnapshot = await userModel
+      .where("roles.in_discord", "==", false)
+      .where("updated_at", ">=", lastTimestamp)
+      .where("updated_at", "<=", currentTimeStamp)
+      .get();
+
+    usersQuerySnapshot.forEach((user) => users.push({ ...user.data(), id: user.id }));
+
+    let orphanTasksUpdatedCount = 0;
+
+    for (const user of users) {
+      const tasksQuerySnapshot = await tasksModel
+        .where("assigneeId", "==", user.id)
+        .where("status", "not-in", ["COMPLETED", "BACKLOG"])
+        .get();
+      tasksQuerySnapshot.forEach(async (taskDoc) => {
+        orphanTasksUpdatedCount++;
+        await tasksModel.doc(taskDoc.id).update({ status: "BACKLOG" });
```

Comment:
can we use constants, for COMPLETED and BACKLOG?

Reply:
Yes done